agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -241,4 +270,32 @@ private static final void setLasModifiedTime(NotificationInfo notification, Temp
                                                                      TimeConvertUtils.YEAR));
   }
 
+  private static final String getEventCreatorOrModifierUserName(long idUser) {
+    IdentityManager identityManager = ExoContainerContext.getService(IdentityManager.class);
+    Identity identity = identityManager.getIdentity(String.valueOf(idUser));
+    return identity.getProfile().getFullName();
+  }
+
+  private static String getUserAbsoluteURI(String id) {
```

Comment:
the name of the attribute `id` is not explicit, is it a username or an identityId ?

Reply:
ok i will rename to identityId